lib/Raven/Processor.php

```diff
@@ -31,6 +31,7 @@ public function __construct(\Raven\Client $client)
      * Override the default processor options.
      *
      * @param array $options Associative array of processor options
+     * @codeCoverageIgnore
```

Comment:
This is not really needed as the whole method will be removed after #472 will be merged